xlators/mgmt/glusterd/src/glusterd-handler.c

```diff
@@ -3763,6 +3771,8 @@ glusterd_deprobe_begin(rpcsvc_request_t *req, const char *hoststr, int port,
     glusterd_peerinfo_t *peerinfo = NULL;
     glusterd_friend_sm_event_t *event = NULL;
     glusterd_probe_ctx_t *ctx = NULL;
+    xlator_t *this = THIS;
```

Comment:
nit: this function (glusterd_deprobe_begin) could be made static. It is only called twice, both from __glusterd_handle_cli_deprobe() - which already has 'this = THIS' - so you can just pass it along.